misc/scripts/deploy-tools.sh

```diff
@@ -222,7 +222,7 @@ function cleanvm {
 function cleanupVMPre {
     case $FILE_EXT in
     deb)
-        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null"
+        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null 2>&1"
```

Comment:
That does not explain failure to stat existing files. E.g. confg.json. it also does not explain the message from tar about time in the future.  CI race does explain both